irohad/model/converters/impl/pb_query_factory.cpp

```diff
@@ -15,19 +15,34 @@
  * limitations under the License.
  */
 
-#include <model/model_hash_provider_impl.hpp>
+#include "model/common.hpp"
 #include "model/converters/pb_query_factory.hpp"
+#include "model/model_hash_provider_impl.hpp"
 #include "model/queries/get_account.hpp"
 #include "model/queries/get_account_assets.hpp"
 #include "model/queries/get_signatories.hpp"
 #include "model/queries/get_transactions.hpp"
 
+
 namespace iroha {
   namespace model {
     namespace converters {
 
-      std::shared_ptr<model::Query> PbQueryFactory::deserialize(
-           const protocol::Query &pb_query) {
+      PbQueryFactory::PbQueryFactory() {
+        log_ = logger::log("PbQueryFactory");
```

Comment:
Need to check handler map with class registrator for queries. Look it in `irohad/model/registration/query_registration.hpp`.